app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,31 @@
 
   <body>
     <nav class = "topnav">
-      <%= link_to "All Merchants", "/merchants"%>
-      <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register", "/register"%>
+      <%= link_to "Home", "/" %>
+      <%= link_to "All Merchants", "/merchants" %>
+      <%= link_to "All Items", "/items" %>
+      <% if user == nil %>
+        <%= link_to "Cart: #{cart.total_items}", "/cart" %>
+        <%= link_to "Log In", "/login" %>
+        <%= link_to "Register", "/register" %>
+      <% elsif user.role == "regular" %>
```

Comment:
The `User` model uses the `enum` method to assign the `role` attribute

That means we get a bunch of methods for free, when we want to check
`user.role == "regular"` === `user.regular?`